JuliaGPU/session1/Manav Singhal/q1.jl

```diff
@@ -0,0 +1,25 @@
+function dectobin(no)
```

Comment:
You could try bit manipulation instead of this. By doing so, you can extend the range of input to 64-bit numbers.